src/pfe/file-watcher/server/test/functional-test/configs/app.config.ts

```diff
@@ -28,14 +28,15 @@ export const projectConfigs = {
     "appDirectory": path.join(codewindWorkspaceDir, projectPrefix)
 };
 
-export const projectTypes = ["docker", "nodejs"];
+export const projectTypes = ["docker", "nodejs", "spring"];
 export const supportedDockerProjects = ["go", "lagom", "python"];
 
 export const templateNames: any = {
     "go": "microclimategotemplate",
     "lagom": "lagomjavatemplate",
     "python": "svtpythontemplate",
-    "nodejs": "node"
+    "nodejs": "node",
+    "spring": "springjavatemplate"
```

Comment:
Is there a way around to not hardcode these names? Will our tests break if template name changes?

Reply:
it is actually not the real template name. it is the folder under `<project_name>/chart`. i don't know how we can get it dynamically. if you have a better idea let me know and I am more than willing to make it dynamic

Reply:
Not sure about that. We can discuss it later and think of a solution if needed. For now it's ok .